flags/flags.go

```diff
@@ -445,7 +445,8 @@ func ParseTemplateData(ctx *cli.Context) (json.RawMessage, error) {
 //  - Error if the URL scheme is not implicitly or explicitly 'https'.
 func ParseCaURL(ctx *cli.Context) (string, error) {
 	caURL := ctx.String("ca-url")
-	if caURL == "" {
+	if caURL == "" && 
+           !ctx.IsSet("offline") {
```

Comment:
I would use `!ctx.Bool("offline")` because --offline=false is a valid option too.